tsconfig.json

```diff
@@ -17,6 +17,9 @@
     "jsx": "react-jsx",
     "downlevelIteration": true,
     "baseUrl": "./src",
+    "paths": {
+      "utils": ["utils/index"]
+    },
     "resolveJsonModule": true,
     "sourceMap": true
```

Comment:
Hmm if we want to disable sourcemaps, we should change this line instead right?

Reply:
resolving this, since we want to enable sourcemaps